bundles/org.openhab.binding.siemenshvac/README.md

```diff
@@ -0,0 +1,129 @@
+A typical system is composed of:
+         
+![Diagram](doc/Diagram.png)                 
+
+There's a lot of different HVAC controllers depending on model in lot of different PAC constructors.
+Siemens RVS41.813/327 inside a Atlantic Hybrid Duo was used for the developpement, and is fully supported and tested.
+
+Siemens have a complete set of controller references under the name "Siemens Albatros".
+Here is a picture of such device.
+You can also find this device in other types of heating systems: boiler or solar based.
+
+![](doc/Albatros.jpg)
+
+You will find some information about the OZW672.01 gateway on the Siemens web site: 
+
+[OZW 672 Page](https://hit.sbt.siemens.com/RWD/app.aspx?rc=FR&lang=fr&module=Catalog&action=ShowProduct&key=BPZ:OZW672.01)
+
+With this binding, you will be able to:
+
+- Consult the different parameters of your system, like temperature, current heating mode, water temperature, and many more.
+- Modify the functioning mode of your device: temperature set point, heating mode, and others.
+
+## Supported Things
+
+Support many different things as the thing type is handle by autodiscovery.
+
+Mainly, it will first discover the gateway.
+Currently test and support is the OZW672.x series.
+No test done with OZW772.x series, but it should work as well.
+
+After, it will discover thing inside your PAC, mainly main controller of type RVS...
+Only test in real condition with RVS41.813/327 but should work with all other type as the access interface is standard.
+
+
+## Discovery
+
+Discovery of Gateway can be done using Upnp.
+Just switch off/on your gateway to make it annonce itself on the network.
+The gateway should appears in the Inbox a few minutes after.
+Be aware what you will have to modifity the password in Gateway parameters just after the discovering to make it work properly.
+Be also aware that first initialization is a little long because we need to read all the metadata from the device.
+
+Discovery of Hvac device have to be done through the Scan button inside the binding.
+Go to the thing page, click on the "+" button, select the siemensHvac binding, and then click Scan.
+Your device should appears on the page after a few seconds.
+
+
+## Binding Configuration
+
+There is no particular configuration to be done.
+The only revelant parameters is on the OZW672 thing for the user and password to use to connect to the gateway.
+IP should have be discovered automatically via UPNP.
+
+
+## Bridge Configuration
+
+Parameter       | Required       | Default        | Description
+----------------|----------------|----------------|------------------
+baseUrl         | yes            |                | The address of the OZW672 devices
+userName        | yes            | Administrator  | The user name to log into the OZW672
+userPass        | yes            |                | The user password to log into the OZW672
+
+
+
+## Thing Configuration
+
+
+
```

Comment:
Sorry, remove it

Reply:
👍 It might become relevant again though, depending on the [thing type discussion](https://github.com/openhab/openhab-addons/pull/14263#discussion_r1598335493), since you should be able to identify the device from the Thing's configuration rather than something embedded into the Thing Type ID or properties.